frontend/src/integrations/discourse/components/discourse-connect-drawer.vue

```diff
@@ -0,0 +1,421 @@
+                <el-button
+                  class="append-icon"
+                  :disabled="!isAPIConnectionValid"
+                  @click="copyToClipboard('url')"
+                >
+                  <i class="ri-file-copy-line" />
+                </el-button>
+              </el-tooltip>
+            </template>
+          </el-input>
+        </app-form-item>
+        <app-form-item label="Webhook Secret">
+          <el-input
+            ref="inputRef"
+            :value="webhookSecret"
+            readonly
+            :disabled="!isAPIConnectionValid"
+            :type="showToken ? 'text' : 'password'"
+          >
+            <template #append>
+              <el-tooltip
+                v-if="!showToken"
+                content="Show Webhook Secret"
+                placement="top"
+              >
+                <el-button
+                  class="append-icon"
+                  :disabled="!isAPIConnectionValid"
+                  @click="onShowToken"
+                >
+                  <i class="ri-eye-line" />
+                </el-button>
+              </el-tooltip>
+              <el-tooltip
+                v-else
+                content="Copy to clipboard"
+                placement="top"
+              >
+                <el-button :disabled="!isAPIConnectionValid" @click="copyToClipboard('secret')">
+                  <i class="ri-file-copy-line" />
+                </el-button>
+              </el-tooltip>
+            </template>
+          </el-input>
+        </app-form-item>
+      </el-form>
+      <el-card v-if="isAPIConnectionValid && props.integration?.settings?.forumHostname" shadow="never" class="rounded-[6px]">
+        <div class="mb-3 flex flex-row w-full justify-between">
+          <el-button :disabled="isWebhookVerifying" class="btn btn--bordered" @click="verifyWebhook()">
+            Verify webhook
+          </el-button>
+          <div v-if="isWebhookVerifying == null" />
+          <div v-else-if="isWebhookVerifying">
+            <i class="ri-loader-4-line animate-spin text-gray-900 w-[14px] mr-2" />
+            <span class="text-gray-900 text-[13px]">Verifying</span>
+          </div>
+          <div v-else-if="isWebhookValid">
+            <i class="ri-check-line text-green-500 w-[14px] mr-2" />
+            <span class="text-green-500 text-[13px]">Succesfully verified</span>
+          </div>
+          <div v-else>
+            <i class="ri-error-warning-line text-red-500 w-[14px] mr-2" />
+            <span class="text-red-500 text-[13px]">No webhooks received yet</span>
+          </div>
+        </div>
+        <div
+          class="text-2xs text-gray-500 leading-normal mb-1"
+        >
+          Confirm if your webhooks are properly configured and crowd.dev is receiving data from Discourse.
+        </div>
+      </el-card>
+    </template>
+
+    <template #footer>
+      <div style="flex: auto">
+        <el-button
+          class="btn btn--md btn--transparent mr-3"
```

Comment:
The class should be `btn--bordered` instead of `btn--transparent`, to match the other drawers